acct-group/sqlgrey/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>gentoo_bugs_peep@parallaxshift.com</email>
+    <name>Philippe Chaintreuil</name>
+  </maintainer>
+
+  <maintainer type="person">
+    <email>mjo@gentoo.org</email>
```

Comment:
I'm confused. Did mjo request being added here or agree on being added?

Reply:
Good catch.  That's a copy-and-paste from another package I maintain.

Do I need an actual Gentoo dev in metadata.xml, or is it just fine with me + proxy-maint?

Reply:
Just you and us are fine.

Reply:
👍 Thanks.